Python/Tests/TestData/DebuggerProject/AttachOutput.py

```diff
@@ -1,6 +1,8 @@
 import sys
 
-while type(sys.stdout).__name__ != '_DebuggerOutput': pass
+attached = False
+while not attached:
+    pass

```

Comment:
Maybe a comment explaining how this works or where to look? Just in case someone comes along and wants to reuse this .py file for another test.